chef_master/source/server_security.rst

```diff
@@ -310,6 +310,105 @@ These newer add-ons will also write all of their secrets to ``/etc/opscode/priva
 
 While the file does not contain passwords in plaintext, it is not safe to share with untrusted users. The format of the secrets file allows Chef Infra Server deployments to conform to regulations that forbid the appearance of sensitive data in plain text in configuration files; however, it does not make the file meaningfully more secure.
 
+SSL Encryption Between Chef Infra Server and External PostgreSQL
+================================================================
+
+**New in Chef Server X.Y:**  Chef Server X.Y introduces the capability to encrypt traffic between Chef Infra Server and an external PostgreSQL configuration over SSL.  These instructions are not all-encompassing and assume familiarity with PostgreSQL administration, configuration, and troubleshooting.  Consult the appropriate PostgreSQL documentation as necessary, e.g. https://www.postgresql.org/docs/9.6/ssl-tcp.html.
+
+To enable this encryption (typical scenario):
+
+1. Ensure that OpenSSL is installed on the PostgreSQL machine.
```

Comment:
might be useful to include the commands to check that? something like `dpkg -l | grep -i openssl` or `rpm -qa | grep -i openssl`

Reply:
Added reference to OpenSSL documentation.